content/posts/2025-07-17-whats-new-1.9/index.md

```diff
@@ -0,0 +1,197 @@
+ovn-ipsec-containerized-w6r9c            1/1     Running   1             4m14s
+ovn-ipsec-host-27j49                     2/2     Running   2             4m14s
+ovn-ipsec-host-jjb54                     2/2     Running   2             4m14s
+ovnkube-control-plane-595bf6d946-9gjjb   2/2     Running   6             5h24m
+ovnkube-node-dfhck                       7/8     Running   7 (16s ago)   3m32s
+ovnkube-node-dtq28                       7/8     Running   8             4m11s
+```
+
+However, Network Observability will report an "AxiosError" if using Loki. Wait at least another 10 minutes or more until the service CA (Certificate Authority) is updated.
+
+
+## Flowlogs-pipeline filter
+
+Flowlogs-pipeline filter lets you filter data after it has been enriched with Kubernetes information.  It can filter logs (Loki data), metrics (Prometheus data), or logs to be exported, which corresponds to the **outputTarget** field (Figure 4) when configuring a FlowCollector instance.  In the FlowCollector form view, scroll down to the **Processor configuration** section and click to open it.  Then click **Filters** and then **Add Filters**.  You can also set a different sampling value than the one used by the eBPF Agent.
+
+The query uses a simple query language that supports 8 comparison operators, the 6 standard ones (`=`, `!=`, `>`, `>=`, `<`, `<=`), plus two more to match or not match a regular expression (`=~`, `!~`).  It can check if a field exists or not (`with(field)`, `without(field)`.  Finally, the query language allows `and` and `or` with parentheses to specify precedence for more complex expressions.
+
+![Flowlogs-pipeline filters config](flowlogs-filters-config.png)<br>
+Figure 4: Flowlogs-pipeline filters configuration
+
+Don't confuse this with the eBPF flow filter, which happens at a much earlier stage at the packet level.  Flowlogs-pipeline filter doesn't benefit as much from a resource savings as eBPF flow filter because part of the processing of flows has already happened.  To get a list of field names for the query, click a row in the Traffic flows table and then click the **Raw** tab.
+
+Here's a query to include only "netobserv" traffic.
+
+```
+SrcK8S_Namespace="netobserv" or DestK8S_Namespace="netobserv"
+```
+
+For more information, see [FLP filtering language](https://github.com/netobserv/flowlogs-pipeline/blob/main/docs/filtering.md).
+
+
+## UDN Mapping (GA)
+
+The eBPF feature **UDNMapping** reached General Availability (GA).  Network Observability added support for the ClusterUserDefinedNetwork object.  In 1.8, it only supported UserDefinedNetwork.  ClusterUserDefinedNetwork allows pods in different namespaces to communicate with each other using the cluster UDN.  To learn more about how to set up a UDN, see the article [User defined networks in Red Hat OpenShift Virtualization](https://www.redhat.com/en/blog/user-defined-networks-red-hat-openshift-virtualization).
+
+
+## Network Observability CLI enhancements
```

Comment:
Do you want to cover all the new features or just pick some ?

https://issues.redhat.com/browse/NETOBSERV-2064

You may mention [metrics](https://github.com/netobserv/network-observability-cli/pull/289) and [pcapng improvments](https://github.com/netobserv/network-observability-cli/pull/293) on top of the query, sampling and yaml already addressed here.